### DISCUSSION TITLE
GPU for experimental SRGAN shaders

### DISCUSSION THREAD

**Discussion by duelistgamer** [bloc97/Anime4K#168] [Q&A]
I wanted to try out the new experimental SRGAN shaders, but I understand that the commit message says you need a powerful GPU.

What GPU should I get if I have older or low res sources that I want to try it on? I have a 7970 currently to spare but obviously it is not enough.

Will a 3060 or 6600XT do the job?

**Comment by bloc97** [comment]
Anything similar in performance or more than a Vega 64 should be more than capable when using the shaders on 360p video. Is it crashing when you try with a Radeon 7970? It might have enough performance, but just barely.

**Reply by duelistgamer** [reply]
I haven't noticed any crashes because I haven't used it yet, but I wanted to try them on both 360p and higher resolution content where the new shaders might benefit the upscaling process.

The main thing is my 7970 is a spare GPU and I don't have a system to build it around. Since I was going to buy new components anyway, I figured I should find out which GPU to get as a baseline for 360/480/720p -> 2160p conversions.
